LBHFSSPublicAPI/Startup.cs

```diff
@@ -42,9 +42,6 @@ public static void ConfigureServices(IServiceCollection services)
                 .AddMvc(setupAction =>
                 {
                     setupAction.EnableEndpointRouting = false;
-                }).AddJsonOptions(jsonOptions =>
-                {
-                    jsonOptions.JsonSerializerOptions.PropertyNamingPolicy = null;
                 })
```

Comment:
This was not mentioned in the description - should this change be there?

Reply:
Oh, never mind. Just noticed the last commit. So do I understand this correctly: does removing this, make json serializer go back to default options, which by default ensure that property names start with lower case?

Reply:
Ahh I missed that.  I'll update the PR.

Reply:
Yes you are correct.  It is not there by default.  It must have been added in while the API was being set up.  The default is camel case and not Pascal case as is used in the classes.